<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>689</width>
    <height>413</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>80</horstretch>
    <verstretch>60</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>300</width>
    <height>255</height>
   </size>
  </property>
  <property name="mouseTracking">
   <bool>false</bool>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::CustomContextMenu</enum>
  </property>
  <property name="windowTitle">
   <string>Appendix</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>100</width>
     <height>100</height>
    </size>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QSplitter" name="splitter_2">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <widget class="QSplitter" name="splitter_4">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <widget class="QSplitter" name="splitter">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <widget class="QTabWidget" name="tabWidget">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>100</horstretch>
           <verstretch>255</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>300</width>
           <height>0</height>
          </size>
         </property>
         <property name="currentIndex">
          <number>-1</number>
         </property>
         <property name="tabsClosable">
          <bool>true</bool>
         </property>
         <property name="movable">
          <bool>false</bool>
         </property>
        </widget>
       </widget>
       <widget class="QWidget" name="layoutWidget">
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QTextEdit" name="textEdit">
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit"/>
         </item>
        </layout>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>689</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Файл</string>
    </property>
    <addaction name="action"/>
    <addaction name="action_2"/>
    <addaction name="action_4"/>
    <addaction name="action_3"/>
    <addaction name="action_5"/>
    <addaction name="action_6"/>
    <addaction name="action_7"/>
    <addaction name="separator"/>
    <addaction name="action_9"/>
    <addaction name="separator"/>
    <addaction name="action_11"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>Правка</string>
    </property>
    <addaction name="action_13"/>
    <addaction name="action_14"/>
    <addaction name="action_15"/>
    <addaction name="separator"/>
    <addaction name="action_17"/>
    <addaction name="action_18"/>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string>Вид</string>
    </property>
    <addaction name="action_19"/>
    <addaction name="action_20"/>
    <addaction name="action_21"/>
    <addaction name="separator"/>
    <addaction name="action_23"/>
   </widget>
   <widget class="QMenu" name="menu_4">
    <property name="title">
     <string>Программа</string>
    </property>
    <widget class="QMenu" name="menu_6">
     <property name="title">
      <string>Сменить язык</string>
     </property>
     <addaction name="actionPascal"/>
     <addaction name="actionC"/>
     <addaction name="actionAppendix"/>
    </widget>
    <addaction name="action_24"/>
    <addaction name="action_25"/>
    <addaction name="separator"/>
    <addaction name="menu_6"/>
    <addaction name="action_8"/>
   </widget>
   <widget class="QMenu" name="menu_5">
    <property name="title">
     <string>Помощь</string>
    </property>
    <addaction name="action_27"/>
    <addaction name="action_28"/>
    <addaction name="action_29"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
   <addaction name="menu_3"/>
   <addaction name="menu_4"/>
   <addaction name="menu_5"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="iconSize">
    <size>
     <width>35</width>
     <height>24</height>
    </size>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="action"/>
   <addaction name="action_2"/>
   <addaction name="action_4"/>
   <addaction name="action_3"/>
   <addaction name="action_24"/>
   <addaction name="action_25"/>
  </widget>
  <action name="action">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/res/ico/new.ico</normaloff>:/res/ico/new.ico</iconset>
   </property>
   <property name="text">
    <string>Новый</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="action_2">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/res/ico/open.ico</normaloff>:/res/ico/open.ico</iconset>
   </property>
   <property name="text">
    <string>Открыть</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="action_3">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/res/ico/save_as.ico</normaloff>:/res/ico/save_as.ico</iconset>
   </property>
   <property name="text">
    <string>Сохранить как...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="action_4">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/res/ico/save.ico</normaloff>:/res/ico/save.ico</iconset>
   </property>
   <property name="text">
    <string>Сохранить</string>
   </property>
  </action>
  <action name="action_5">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/res/ico/save_all.ico</normaloff>:/res/ico/save_all.ico</iconset>
   </property>
   <property name="text">
    <string>Сохранить всё</string>
   </property>
  </action>
  <action name="action_6">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/res/ico/close.ico</normaloff>:/res/ico/close.ico</iconset>
   </property>
   <property name="text">
    <string>Закрыть</string>
   </property>
  </action>
  <action name="action_7">
   <property name="text">
    <string>Закрыть неактивные</string>
   </property>
  </action>
  <action name="action_9">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/res/ico/print.ico</normaloff>:/res/ico/print.ico</iconset>
   </property>
   <property name="text">
    <string>Печать</string>
   </property>
  </action>
  <action name="action_11">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/res/ico/exit.ico</normaloff>:/res/ico/exit.ico</iconset>
   </property>
   <property name="text">
    <string>Выход</string>
   </property>
  </action>
  <action name="action_13">
   <property name="text">
    <string>Вырезать</string>
   </property>
  </action>
  <action name="action_14">
   <property name="text">
    <string>Копировать</string>
   </property>
  </action>
  <action name="action_15">
   <property name="text">
    <string>Вставить</string>
   </property>
  </action>
  <action name="action_17">
   <property name="text">
    <string>Найти</string>
   </property>
  </action>
  <action name="action_18">
   <property name="text">
    <string>Заменить</string>
   </property>
  </action>
  <action name="action_19">
   <property name="text">
    <string>Окно вывода</string>
   </property>
  </action>
  <action name="action_20">
   <property name="text">
    <string>Окно ввода</string>
   </property>
  </action>
  <action name="action_21">
   <property name="text">
    <string>Окно отладки</string>
   </property>
  </action>
  <action name="action_23">
   <property name="text">
    <string>Очистить вывод</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Del</string>
   </property>
  </action>
  <action name="action_24">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/res/ico/play.ico</normaloff>:/res/ico/play.ico</iconset>
   </property>
   <property name="text">
    <string>Выполнить</string>
   </property>
   <property name="shortcut">
    <string>F9</string>
   </property>
  </action>
  <action name="action_25">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/res/ico/stop.ico</normaloff>:/res/ico/stop.ico</iconset>
   </property>
   <property name="text">
    <string>Завершить</string>
   </property>
  </action>
  <action name="action_27">
   <property name="text">
    <string>Справка</string>
   </property>
   <property name="shortcut">
    <string>F1</string>
   </property>
  </action>
  <action name="action_28">
   <property name="text">
    <string>Примеры</string>
   </property>
  </action>
  <action name="action_29">
   <property name="text">
    <string>Об авторе...</string>
   </property>
  </action>
  <action name="action_8">
   <property name="text">
    <string>Настройки</string>
   </property>
  </action>
  <action name="actionPascal">
   <property name="text">
    <string>Pascal</string>
   </property>
  </action>
  <action name="actionC">
   <property name="text">
    <string>C++</string>
   </property>
  </action>
  <action name="actionAppendix">
   <property name="text">
    <string>Appendix</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
